recipe/meta.yaml

```diff
@@ -12,10 +12,10 @@ source:
     - 0001-Remove-assertions-on-longer-than-sockaddr_un.sun_pat.patch
 
 build:
-  number: 0
+  number: 1
   skip: true  # [not linux]
   run_exports:
-    - {{ pin_subpackage("rdma-core") }}
+    - {{ pin_subpackage("rdma-core", max_pin="x") }}
```

Comment:
This is [the default behavior for `pin_subpackage`]( https://docs.conda.io/projects/conda-build/en/latest/resources/variants.html#extra-jinja2-functions )

So this change isn't strictly needed

Reply:
Thanks John, in the interest of not generating a new redundant package, I've opened a draft PR https://github.com/conda-forge/rdma-core-feedstock/pull/10 so that we can revert this change when we do a future update here.